proto/vtctldata.proto

```diff
@@ -127,6 +127,8 @@ message Workflow {
     // if there are N logs that exist for the stream, and we fail to fetch the
     // ith log, we will still return logs in [0, i) + (i, N].
     string log_fetch_error = 14;
+    // Note that there is a single tags column: multiple tags can be "stuffed" into this as a CSV, for example
+    string tags = 15;
```

Comment:
imo this should be `repeated string tags = 15`, and then we do the conversion once in workflow/server from csv string to string slice, and then everyone downstream from that gets the nicely-structured list rather than needing to re-split

Reply:
Good idea: done.